src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(pyxtp LANGUAGES CXX)
+
+find_package(pybind11 REQUIRED HINTS)
+find_package(VOTCA_XTP REQUIRED NO_MODULE)
+
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds
+ PUBLIC
+ ${PROJECT_SOURCE_DIR}/binds/include
+)
+
+target_compile_options(xtp_binds
+ PUBLIC
+ "-Wall"
+)
+set_target_properties(xtp_binds
+  PROPERTIES
+  CXX_STANDARD 14
+  CXX_STANDARD_REQUIRED ON
+)
+
+
+target_link_libraries(xtp_binds
+ PUBLIC
+ VOTCA::votca_xtp VOTCA::votca_tools
+ "-Wl,--no-as-needed"
```

Comment:
Why is this needed? Better try to fix the code, not all compiler support this option.

Reply:
That was due MKL not finding some symbols

Reply:
Then we should make it conditional on `MKL_FOUND` similar to what we did in `votca/votca`. Also if it is fully integrated we can drop this.